packages/indexer-agent/src/network.ts

```diff
@@ -206,6 +210,7 @@ export class Network {
       paused,
       isOperator,
       restakeRewards,
+      parseGRT(String(queryFeesCollectedClaimThreshold))
```

Comment:
Should we use a more explicit number to string conversion here? I'm not sure what `String()` will do to the number.

Reply:
String() is a essentially a wrapper around `toString` with other fallbacks (for instance if the value is null or not a primitive.) 

As such: either works in this case, but since the threshold value should always be an integer it may be slightly more clear to use the more specific value.toString() conversion.

Reply:
I'll go ahead and update to use `.toString()`.